commit fe309d5f45126a441d0e242e5d34bc75b327c923 (refs/changes/92/4892/1)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-09-27T15:40:02-07:00 (1 year ago)
    
    TRITON-801 net-agent doesn't update NIC state if VM is created but not updated

diff --git a/lib/vmadm-watcher-fsm.js b/lib/vmadm-watcher-fsm.js
index d844945..e1e3745 100644
--- a/lib/vmadm-watcher-fsm.js
+++ b/lib/vmadm-watcher-fsm.js
@@ -90,6 +90,7 @@ VmadmEventsFSM.prototype.state_init = function (S) {
             assert(!self.vms.hasOwnProperty(ev.zonename), 'VM already created');
 
             self.vms[ev.zonename] = ev.vm;
+            needsUpdate = true;
             break;
         case 'modify':
             assert.object(ev.vm, 'ev.vm');
